docs/source/guide/cdr-1-intro.myst

```diff
@@ -0,0 +1,137 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use CDR?
+
+Here we show how to use Clifford data regression (CDR) by means of a simple example. CDR is a learning-based quantum error mitigation technique that relies on the training of a classical simulator with quantum circuits that resemble the one that is the aim of the quantum error mitigation, but which are easier to simulate {cite}`Czarnik_2021_Quantum`. More information about this technique can be found in the section [What is the theory behind CDR?](cdr-5-theory.myst).
+
+```{code-cell} ipython3
+import warnings
+warnings.filterwarnings("ignore")
+
+import numpy as np
```

Comment:
```suggestion
import numpy as np
import warnings
warnings.simplefilter("ignore", np.ComplexWarning)
```

Instead of ignoring all warnings it is better to filter only the one that we need. 

We need this since `Observable` produce expectation values of complex type while CDR is compatible only with real expectation values. I agree that this is not very nice but, given the current structure of the code, its a reasonable compromise.